jupyter_desktop/__init__.py

```diff
@@ -10,14 +10,28 @@ def setup_desktop():
     # This is only readable, writeable & searchable by our uid
     sockets_dir = tempfile.mkdtemp()
     sockets_path = os.path.join(sockets_dir, 'vnc-socket')
+    turbovnc = '/opt/TurboVNC/bin/vncserver'
```

Comment:
Should we just check for it in $PATH maybe? Instead of assuming a hardcoded path? We could also do a version check or something to make sure it isn't a random binary named vncserver.

Reply:
Done. I didn't add a version/name check or anything else since it's possible people will want to use other vncserver implementations.